packages/framework-provider-azure-infrastructure/test/infrastructure/stacks/schedules-functions.test.ts

```diff
@@ -0,0 +1,117 @@
+import { expect } from '../../expect'
+import { SchedulesFunctions } from '../../../src/infrastructure/stacks/schedules-functions'
+import { BoosterConfig, ScheduledCommandInterface, ScheduleInterface } from '@boostercloud/framework-types'
+import { describe } from 'mocha'
+import { ScheduleFunctionDefinition } from '../../../src/infrastructure/types/functionDefinition'
+
+describe('Creating schedules-functions', () => {
```

Comment:
This should be **scheduleD** instead of **scheduleS**, no?

Reply:
ouch haha, thanks